tests/scripts/all.sh

```diff
@@ -1419,9 +1419,24 @@ component_build_crypto_full () {
 component_build_crypto_baremetal () {
   msg "build: make, crypto only, baremetal config"
   scripts/config.py crypto_baremetal
-  make CFLAGS='-O1 -Werror'
+  make CFLAGS="-O1 -Werror -I$(pwd)/tests/include/baremetal-override/"
```

Comment:
Other places in `all.sh` use `$PWD` rather than `$(pwd)` - consistency is good, so is calling `pwd` necessary?